src/runtime/types.ts

```diff
@@ -141,7 +141,7 @@ export interface RuntimeCompatibilitySchema {
   resvg: 'node' | 'wasm' | 'wasm-fs' | false
   satori: 'node' | 'wasm' | 'wasm-fs' | false
   sharp: 'node' | false
-  wasm?: WasmOptions
+  wasm?: NitroOptions['wasm']
```

Comment:
not related to this PR, but unwasm is explicitly a built-time tool same as `NitroOptions`.